src/sign-in-webapp/README.md

```diff
@@ -67,46 +70,143 @@ Later the web app can make calls to the Microsoft 365 Graph API on behalf of sig
    az ad app update --id $AZURE_AD_APP_CLIENT_ID_WEBAPP --set logoutUrl="https://localhost:5001/signout-oidc"
    ```
 
-## Scaffold the web app by using the ASP.NET Core Razor project template
+</details>
 
-1. Install the latest `Microsoft.Identity.Web` templates from NuGet.org
+### 2. Configure the web app
 
-   ```bash
-   dotnet new --install Microsoft.Identity.Web.ProjectTemplates::1.21.0
-   ```
+1. Open the `WebApp.csrpoj` under the the `sign-in-webapp` folder in your code editor.
```

Comment:
Somewhat of a nit, but since there will be potentially hundreds of file references, let's use the Docs style guide for text formatting.

Here are some things we'll use frequently in these sample READMEs. Will add to our code sample style guide.

| Formatted example                      | Description                                        |
|----------------------------------------|----------------------------------------------------|
| _~/samples/python-web-app/settings.py_ | Files, directories, paths                          |
| **Save**, **Application (client) ID**  | UI controls (including labels)                     |
| `PublicClient`, `msal-browser`, `int`  | Code elements like types, namespaces, packages, .. |
| _MyAwesomeWebApp_                      | Text entered by the user                           | 

> NOTE: I'm not a fan of _italics_ for text entered by the user (I prefer `code` formatting), but since these READMEs will light up on Docs, we need to stick to the Docs guidelines.

Reply:
thanks for the feedback @mmacy,

done | addressed from [49962](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/49962027343c860f982691aeb1f2fd7e2ad05fbb)

Reply:
Marking resolved.